README.md

```diff
@@ -132,6 +132,8 @@ Differences are noted here:
 |`processArguments`|Process arguments and environment which will be sent to the `WebDriverAgent` server.|`{ args: ["a", "b", "c"] , env: { "a": "b", "c": "d" } }` or `'{"args": ["a", "b", "c"], "env": { "a": "b", "c": "d" }}'`|
 |`autoLaunch`|When set to `false`, prevents the application under test from being launched automatically as a part of the new session startup process. The launch become the responsibility of the user. Defaults to `true`.|`true` or `false`|
 |`allowProvisioningDeviceRegistration`|Allow `xcodebuild` to register your destination device on the developer portal if necessary. Requires a developer account to have been added in Xcode's Accounts preference pane. Defaults to `false`.|`true` or `false`|
+|`resultBundlePath`| Specify the path to the result bundle path as `xcodebuild` argument for `WebDriverAgent` build. `WebDriverAgent` process must start/stop every time to set this path properly. `useNewWDA` may help then. Please read `xcodebuild` command help for more details. | e.g. `/path/to/resultbundle` |
```

Comment:
`to set this path properly` - I would rather say `to pick up changed value of this property`